### PR TITLE
Addition of draft text announcing sunset of 613

### PR DESCRIPTION
**Pull request by mathew-thomson**
Added draft text in both languages to the localization csv. I then added the tag for that text chunk to top of the index and wastewater pages.

@DougManuel - please review for content, @rvyuha  - please review for functionality.

### CODE REVIEW

**Comment by rvyuha** [comment]
I was experimenting with the idea of a warning box in this [PR](https://github.com/Big-Life-Lab/Ottawa-COVID-Projection/pull/106). @mathew-thomson What do you think of putting some of the text in a warning box? Or would a simple paragraph be better?

File: Data/translations/Covid Localization - Home Page.csv [hunk 1]
```diff
@@ -153,12 +153,25 @@ definitions_link,## {#defintions},## {#defintions}
 DT_label_double,doubling time,temps de doublement
 DT_label_half,halving time,temps de division
 DT_label_no_change,no growth (flat),pas de croissance
+end_transition_out,"**Note:** June 1, 2023
+
+613covid as a public-facing tool has been a point of tremendous pride for our lab. We are very happy that we were able to generate something quickly, with the support and collaboration of our colleagues across disciplines in Ottawa and Ontario. The website has continued to report data on the SARS-CoV-2 signal in wastewater for several years, and has expanded to even include reporting of influenza and RSV viruses. We are so honoured to have so many people in Ottawa use, consult, and support this application of ours.
```
Comment (DougManuel) [inline Data/translations/Covid Localization - Home Page.csv:158]: 'tremendous pride for the Ottawa Covid-19 collaboration between the uOttawa Delatolla lab, the Ottawa Hospital, CHEO and Ottawa Public Health.... We are so honoured to have so many people in Ottawa use, consult, and support this application of ours. There have been over 2 million users.

**Review by DougManuel — COMMENTED** [review]
Let's make sure Rob's lab is at the centre of pride.

**Comment by mathew-thomson** [comment]
Thanks for the suggestion, @rvyuha ! I think now because we're announcing that the site is transitioning, not being terminated, that a warning box might be a bit overkill. But I defer to you! If you think that it's not a "scary" warning box, and that it will gather the attention properly, we can definitely do that. The text is now finalized, so we're good to go whenever we decide.

File: content/en/wastewater.Rmd [hunk 2]
```diff
@@ -81,7 +81,7 @@ var_col <- "rgb(85, 151, 217)"
 n1_col <- "rgb(23, 63, 95)"
 n2_col <- "rgb(60, 174, 163)"
 n1_n2_col <- "rgb(237, 85, 59)"
-rolling_avg_col <- "rgb(226, 127, 88)"
+rolling_avg_col ¿)"
```
Comment (rvyuha) [inline content/en/wastewater.Rmd:84]: Was this an intentional change?

File: content/en/_index.Rmd [hunk 3]
```diff
@@ -34,7 +33,10 @@ numofweeks <- "twoweek"
 site <- "CovidEmployeeData"
 ```
 
+`r end_transition_out`
```
Comment (rvyuha) [inline content/en/_index.Rmd:36]: Is this the text edits you made in the csv? If yes they need to be pulled from the localization using the "insert_translation" function.

File: content/en/wastewater.Rmd [hunk 4]
```diff
@@ -401,23 +402,24 @@ p <- reworked_figure(xaxis = "date", yaxis = list(daily_rsv_signal, rsv_7_day),
 p <- layout(p, yaxis = list(range = c(0, 0.0008500), exponentformat="none", tickformat=".5f"))
 p
 ```
-```
+
+\`\`\`
```
Comment (rvyuha) [inline content/en/wastewater.Rmd:406]: What is the purpose of this?

**Review by rvyuha — COMMENTED** [review]
It requires some minor additional changes to make it functional. In addition, should we update the fr pages too?

File: content/en/wastewater.Rmd (hunk 2 above)
Comment (mathew-thomson) [reply content/en/wastewater.Rmd:84]: lol no Idk how this even happened

File: content/en/wastewater.Rmd (hunk 4 above)
Comment (mathew-thomson) [reply content/en/wastewater.Rmd:406]: I have a suspicion this was typed by a cat while I was out of the room

File: content/en/_index.Rmd (hunk 3 above)
Comment (mathew-thomson) [reply content/en/_index.Rmd:37]: Tried to address this in my most recent push

**Pull request merged**